### PR TITLE
updates security specification on datasets to obtain data for gdpr re…

### PR DESCRIPTION
**Pull request by theseyi**
…lated to each dataset

### CODE REVIEW

File: wherehows-web/app/components/dataset-confidential.js [hunk 1]
```diff
@@ -14,15 +14,29 @@ const {
   String: { htmlSafe }
```
Comment (cptran777) [inline wherehows-web/app/components/dataset-confidential.js]: Just pointing out that you are playing a dangerous game here

File: wherehows-web/app/styles/components/dataset-compliance/_compliance-prompts.scss [hunk 2]
```diff
@@ -27,3 +27,25 @@
...
 }
+
+/// Wraps a status notification icon for a given dataset
+.dataset-tag-container {
+  width: 20px;
+  display: inline-block;
+}
+
+/// A notification that a dataset contains a tag
+.dataset-tagged {
```
Comment (cptran777) [inline wherehows-web/app/styles/components/dataset-compliance/_compliance-prompts.scss]: Wouldn't tagged be a state of the dataset in BEM?

Comment (theseyi) [inline wherehows-web/app/styles/components/dataset-compliance/_compliance-prompts.scss]: Kinda, I made them separate components but I should rename this to `some-icon-class-name--tagged`

File: wherehows-web/app/components/dataset-confidential.js [hunk 3]
```diff
@@ -70,15 +84,31 @@ export default Component.extend({
...
    * Creates a lookup table of fieldNames to classification
    *   Also, the expectation is that the association from fieldName -> classification
    *   is one-to-one hence no check to ensure a fieldName gets clobbered
    *   in the lookup assignment
    */
   fieldNameToClass: computed(
-    `${sourceClassificationKey}.confidential.[]`,
-    `${sourceClassificationKey}.limitedDistribution.[]`,
-    `${sourceClassificationKey}.highlyConfidential.[]`,
+    `${sourceClassificationKey}.{confidential,limitedDistribution,highlyConfidential}.[]`,
```
Comment (justYves) [inline wherehows-web/app/components/dataset-confidential.js]: Interesting, I haven't seen this syntax before for computed properties. Is this new?

File: wherehows-web/app/templates/components/dataset-confidential.hbs [hunk 4]
```diff
@@ -26,6 +26,73 @@
...
+    </tr>
+    </thead>
+    <tbody>
+    {{#each-in datasetClassification as |classification props|}}
+      <tr>
+        <td>
+          {{props.label}}
+
+          <span class="dataset-tag-container">
+            {{#if (eq props.value true)}}
```
Comment (justYves) [inline wherehows-web/app/templates/components/dataset-confidential.hbs]: wouldn't a simple truthy  if/else check work here? `{{#if props.value}} .... {{else}}...`

Comment (theseyi) [inline wherehows-web/app/templates/components/dataset-confidential.hbs]: @justYves nah, the value can be `null` in which case I don't  want to show anything.

Ember's `if` template helper checks for truthiness, but I need a strict equality check in this case

File: wherehows-web/app/components/dataset-confidential.js (hunk 3 above)
Comment (theseyi) [inline wherehows-web/app/components/dataset-confidential.js]: Haha, I found this use case while trolling through the Ember code for expandedProperties here: @justYves 
https://github.com/emberjs/ember.js/blob/v2.12.0/packages/ember-metal/lib/expand_properties.js#L26

Comment (justYves) [inline wherehows-web/app/components/dataset-confidential.js]: Neat!

**Pull request merged**